priv/translations/id/LC_MESSAGES/relative_time.po

```diff
@@ -9,7 +9,6 @@
 msgid ""
 msgstr ""
 "Language: id\n"
-"Plural-Forms: nplurals=2; plural=(n != 1);\n"
```

Comment:
Correct this to match the one for id in units.po.

Reply:
Thanks. Because the file wasn't using plurals I had deleted it, but just to be safe I've updated it to match.